Sources/MySQL/Error.swift

```diff
@@ -11,7 +11,7 @@ public enum Error: Swift.Error {
     case inputBind(String)
     case outputBind(String)
     case fetchFields(String)
-    case prepare(String)
+    case prepare(String, UInt32)
```

Comment:
Unfortunately, this would be considered an API breaking change pushing us to 2.0. If there's no other way around it, I'm not inherently against it, but it's something we'd need to be aware of. This would push driver/provider to 2.0 as well. We would be able to maintain Vapor at 1.0 so it's less inherently consequential over all.